.gitignore

```diff
@@ -45,3 +45,4 @@ _build
 .idea
 .vscode
 *~
+.env
```

Comment:
This is a duplicate, I think you meant to add `.venv`.  Can you add it underneath `.env` too?  It'll match a patch I'm hoping to roll out.

Reply:
Oh I misunderstood what you were asking. I thought you wanted me to rename my local version and add `.env`

Reply:
I'll re-review and add this to that.